### PR TITLE
Feature/ieo light

### PR DESCRIPTION
**Pull request by pkhlopiachyi**

### CODE REVIEW

File: src/plugins/ieo/screens/IEODetailsScreen/index.tsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ class IEODetailsContainer extends React.Component<Props> {
                     {ieoDetails ? <IEODetails currentIEO={currentIEO} ieoDetails={ieoDetails} /> : null}
                 </div>
                 <div className="pg-currentIEO-page__product-intiduction">
-                    {ieoDetails ? <IEOProjectIntroduction introduction={ieoDetails.introduction}/> : null}
+                    <IEOProjectIntroduction introduction={ieoDetails && ieoDetails.introduction}/>
```
Comment (josadcha) [inline src/plugins/ieo/screens/IEODetailsScreen/index.tsx]: don't commit changes in this file

File: src/plugins/ieo/styles/themes/light.pcss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.light-mode {
+  --ieo-header-bg: #FFFFFF;
+  --ieo-header-border: #EAEBEE;
+  --border-top: var(--ieo-header-border);
+}
```
Comment (josadcha) [inline src/plugins/ieo/styles/themes/light.pcss]: add empty line

File: src/styles/themes/light.pcss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 .light-mode {
   --main-bg: #fff;
-  --back-bg: #fff;
+  --back-bg: #F0F1F5;
```
Comment (josadcha) [inline src/styles/themes/light.pcss]: are you sure you need to change light colors in src folder?

File: src/plugins/ieo/containers/IEOListElement/IEOListElement.pcss [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
     @media(max-width: 576px) {
-        flex-direction: column;
+      flex-direction: column;
     }
+  }
+
+  &__header {
+    color: var(--header-text);
+  }
 }
```
Comment (josadcha) [inline src/plugins/ieo/containers/IEOListElement/IEOListElement.pcss]: add empty line

File: src/styles/themes/light.pcss (hunk 3 above)
Comment (pkhlopiachyi) [reply src/styles/themes/light.pcss]: yes

**Pull request merged**